.github/workflows/ci-modules.yaml

```diff
@@ -0,0 +1,73 @@
+name: CI - Modules
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'modules/**'
```

Comment:
Will this cause PRs to main to trigger deploys?

Reply:
as intended, yes.  A "deploy" here means merely `aws s3 cp ...`

Reply:
Does this happen on PR open, or PR merge?

Reply:
Oh I see, it pushes on PR open under a different name so you can test the branch

Reply:
that's right. the workflow invoked in two scenarios:
* pushes to `main` (aka PR merges)
* PR sync events where the base branch is `main`

There is ternary-like behavior on L73 that distinguishes this.